mcs/class/System.Xml.Linq/System.Xml.Linq_xtest.dll.sources

```diff
@@ -0,0 +1,23 @@
+../../../external/corefx/src/Common/tests/System/Xml/ModuleCore/*.cs
+../../../external/corefx/src/Common/tests/System/Xml/XmlCoreTest/*.cs
+../../../external/corefx/src/Common/tests/System/Xml/XmlDiff/*.cs
+../../../external/corefx/src/Common/src/System/Diagnostics/CodeAnalysis/ExcludeFromCodeCoverageAssemblyAttribute.cs
+
+../../../external/corefx/src/Common/tests/System/IO/TempFile.cs
+
+../../../external/corefx/src/Common/tests/System/Xml/XPath/XmlDocument/CreateNavigatorFromXmlDocument.cs
+../../../external/corefx/src/Common/tests/System/Xml/XPath/Common/*.cs
+
+../../../external/corefx/src/System.Private.Xml.Linq/tests/axes/*.cs
+../../../external/corefx/src/System.Private.Xml.Linq/tests/events/*.cs
+#../../../external/corefx/src/System.Private.Xml.Linq/tests/misc/*.cs
```

Comment:
Why are the test disabled?

Reply:
There are some tests that the build breaks on. I'm working on resolving that issues. In particular I have errors like this:
`external/corefx/src/System.Private.Xml.Linq/tests/Properties/FunctionalTests.cs(11,26): error CS0101: The namespace 'CoreXml.Test.XLinq' already contains a definition for 'FunctionalTests'
external/corefx/src/System.Private.Xml.Linq/tests/misc/FunctionalTests.cs(19,28): error CS0111: Type 'FunctionalTests' already defines a member called 'RunTests' with the same parameter types
external/corefx/src/System.Private.Xml.Linq/tests/xNodeBuilder/FunctionalTests.cs(22,28): error CS0111: Type 'FunctionalTests' already defines a member called 'RunTests' with the same parameter types
external/corefx/src/System.Private.Xml.Linq/tests/xNodeReader/FunctionalTests.cs(21,28): error CS0111: Type 'FunctionalTests' already defines a member called 'RunTests' with the same parameter types`

It looks like several corefx classes with the same name/members that come to single mono assembly lead to kind of name duplication.

Reply:
As for `/src/Common/src/System/Diagnostics/CodeAnalysis/ExcludeFromCodeCoverageAssemblyAttribute.cs` I've added it intentionally because by default there is an issue related with absence of this attribute. 
I will check one more time and try to resolve it.